models/eos/interfaces/overridden_example_01.txt

```diff
@@ -0,0 +1,40 @@
+
+# Using overridden
+
+# Before state:
+# -------------
+#
+# veos#show running-config | section interface
+# interface Ethernet1
+#    description "Interface 1"
+# !
+# interface Ethernet2
+# !
+# interface Management1
+#    description "Management interface"
+#    ip address dhcp
+# !
+
+- name: Replaces device configuration of all interfaces with provided configuration
```

Comment:
Should this say "Overrides..." instead of Replaces?  In general, I'm struggling to understand the difference between overriding a configuration and replacing it...and I can't figure that out based on the examples here and in the override example file...